core/src/main/java/org/stellar/anchor/util/AssetHelper.java

```diff
@@ -50,28 +50,6 @@ public static boolean isNativeAsset(String assetCode, String assetIssuer) {
     return "native".equals(assetCode) && isEmpty(assetIssuer);
   }
 
-  /**
-   * Returns the asset id in the SEP-38 asset identification format, or null if the asset is not
-   * supported.
-   *
-   * @param assetCode The asset code
-   * @param assetIssuer The asset issuer
-   * @return The asset id in the SEP-38 asset identification format.
-   */
-  public static String getAssetId(String assetCode, String assetIssuer) {
```

Comment:
Quick question: Is this function not used or moved? I remember it is used somewhere...

Reply:
It was used in removed code